git-filter-repo

```diff
@@ -1246,7 +1248,7 @@ class FastExportParser(object):
     aux_info = {'orig_parents': orig_parents,
                 'had_file_changes': had_file_changes}
     if self._commit_callback:
-      self._commit_callback(commit, aux_info)
+      self._commit_callback(commit, aux_info, self._callback_data)
```

Comment:
If we were to do this, I'd prefer that self._callback_data consistently be the second argument, and aux_info can be last.